nautobot/docs/release-notes/version-1.6.md

```diff
@@ -8,6 +8,25 @@ This document describes all new features and changes in Nautobot 1.6.
 
 ### Added
 
+#### Installation Metrics
```

Comment:
Our general pattern in the release overview in the past has been to have the subheading include a link to the issue or PR on github:

```suggestion
#### Installation Metrics ([#4047](https://github.com/nautobot/nautobot/issues/4047))
```